### PR TITLE
Update contextual-survey-metadata.json for pilot test #1

### PR DESCRIPTION
**Pull request by jayoung-lee**
The URL for "More Info" may be update later to `/user-surveys` once https://github.com/flutter/website/pull/8654 is merged

### CODE REVIEW

File: surveys/contextual-survey-metadata.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [
     {
-        "uniqueId": "eca0100a-505b-4539-96d0-57235f816cef",
-        "startDate": "2023-07-01T09:00:00-07:00",
-        "endDate": "2023-08-31T09:00:00-07:00",
-        "description": "Help improve Flutter's release builds with this 3-question survey!",
+        "uniqueId": "a09da33a-39f4-42b7-9aab-02dd3e0d783",
```
Comment (eliasyishak) [inline surveys/contextual-survey-metadata.json:3]: Looks like the test failed for this ID, there should be 12 characters after the last hyphen for it to be valid UUID v4 formatting

Comment (jayoung-lee) [reply surveys/contextual-survey-metadata.json]: Ah, I missed one letter at the end while copying from the source! Thanks for writing the test. Fixed now.

**Comment by eliasyishak** [comment]
I think you may also want to make the conditions array empty since you want to test with everyone right? As it stands right now, each user testing this feature would need to have already sent 1000 events previously, which may not be the case for everyone.

https://github.com/flutter/uxr/pull/113/files#diff-bdf7173921cada97180d013ce22fd3e8c7560bb877eddbb417724eeb29a3ed8cR13-R19

**Review by eliasyishak — APPROVED** [review]
Sweet, LGTM.. you changed the snooze minutes to 24 hours now it seems so you'll be testing this over the span of 2 days?

**Comment by jayoung-lee** [comment]
Hi Elias, thanks for reviewing! 5 minutes makes more sense, actually. Updated!

**Pull request merged**